compression-service/EXML-converter-service/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
+wrapperUrl=https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/0.5.6/maven-wrapper-0.5.6.jar
```

Comment:
Will these dependencies ever change or update? Or do we need to monitor these to ensure the build is security compliant?